### PR TITLE
bf(S3C-3447): Switch to node-schedule

### PR DESCRIPTION
**Pull request by tmacro**
Changed to bring dependecy inline with other projects. Additionally solves S3C-3339 in the process

### CODE REVIEW

**Review by rahulreddy — APPROVED** [review]

File: libV2/tasks/BaseTask.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const cron = require('node-cron');
+const cron = require('node-schedule');
```
Comment (miniscruff) [inline libV2/tasks/BaseTask.js:1]: should we rename the variable?

Comment (rachedbenmustapha) [reply libV2/tasks/BaseTask.js:1]: It's still a cron implementation :)

Comment (miniscruff) [reply libV2/tasks/BaseTask.js:1]: I mean... true. :laughing:

**Review by miniscruff — APPROVED** [review]

**Pull request merged**